Formula/perl.rb

```diff
@@ -57,7 +57,7 @@ def install
 
   def post_install
     if OS.linux?
-      perl_archlib = Utils.safe_popen_read("perl", "-MConfig", "-e", "print $Config{archlib}")
+      perl_archlib = Utils.safe_popen_read(opt_bin/"perl", "-MConfig", "-e", "print $Config{archlib}")
```

Comment:
```suggestion
      perl_archlib = Utils.safe_popen_read(bin/"perl", "-MConfig", "-e", "print $Config{archlib}")
```
`opt_bin` is a symlink which we don't need to rely on for here.

Reply:
`bin` points to a path in the Cellar, so we don't need to rely on `perl` being linked. `opt_bin/"perl"` could point to something else entirely (for example, if a user for some reason already has a `$HOMEBREW_PREFIX/opt/perl/bin/perl`).